app/src/main/java/com/app/worte/worte/WdbEntry.java

```diff
@@ -0,0 +1,17 @@
+package com.app.worte.worte;
+
+public class WdbEntry
```

Comment:
Consider adding methods for parsing/serializing to strings. E.g. `static WdbEntry fromString(...) ` or `toString()`

Reply:
would be a nice approach, but in this case we have to move the knowledge of .wdb file structure from FileSystemOperator class to WdbEntry and this will add some problems for maintenance